apps/functions-subscription/api/internal.yaml

```diff
@@ -46,6 +46,8 @@ paths:
                 $ref: '#/components/schemas/Subscription'
         '401':
           description: Unauthorized
+        '403':
```

Comment:
Do we have to map this kind of error here? https://github.com/pagopa/trial-system/blob/6c18e0a93005ea45e38d3be40a6b849bac395c49/apps/functions-subscription/src/adapters/azure/functions/errors.ts#L5-L6

Reply:
That function converts an error, specifically a domain error, into an HTTP error. We should map the forbidden error if there's a ForbiddenError in the domain layer that we want to convert to a 403 Forbidden HTTP error. 

The 403 error is already in use ([see the middleware here](https://github.com/pagopa/trial-system/blob/3612db4f69c52395131f09a0e7449863a63e8bc5/apps/functions-subscription/src/adapters/azure/functions/middleware.ts#L61)), but it's not mapped in the OpenAPI specification.